cea/optimization/master/evaluation.py

```diff
@@ -215,81 +221,81 @@ def calc_master_to_slave_variables(individual, Qmax, locator, gv, optimization_c
     master_to_slave_vars = slave_data.SlaveData()
     configkey = "".join(str(e)[0:4] for e in individual)
     
-    individual_barcode = sFn.individual_to_barcode(individual, gv, optimization_constants)
+    individual_barcode = sFn.individual_to_barcode(individual)
     configkey = configkey[:-len(individual_barcode)] + hex(int(str(individual_barcode),2))
     master_to_slave_vars.configKey = configkey
     master_to_slave_vars.nBuildingsConnected = individual_barcode.count("1") # counting the number of buildings connected
     
-    Qnom = Qmax * (1+optimization_constants.Qmargin_ntw)
+    Qnom = Qmax * (1+Qmargin_ntw)
     
     # Heating systems
     
     #CHP units with NG & furnace with biomass wet
     if individual[0] == 1 or individual[0] == 3:
-        if optimization_constants.Furnace_allowed == 1:
+        if Furnace_allowed == 1:
```

Comment:
I would really like this to be a boolean... unless... what are valid values for this variable?